src/swig/python2/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+
+#
+# FIXME: the find_package functions climbs up to loading python3,
+#        cannot find a way around it - setting manual paths instead
+# find_package(PythonLibs 2.7 EXACT REQUIRED)
+#
+
+if (${CMAKE_VERSION} VERSION_GREATER_EQUAL 3.12)
```

Comment:
@skarbat I've put in this CMake version check before trying to hardcode a Python 2 path because otherwise it will never build on anything other than arm.